packages/x-license/LICENSE.md

```diff
@@ -1,7 +1,7 @@
-# [Material-UI X](https://material-ui.com/x/)
+# [Material-UI X Pro](https://material-ui.com/x/)
```

Comment:
Actually, this might be even clearer

```suggestion
# [Material-UI X Pro](https://material-ui.com/pricing/)
```

Reply:
Yeah, maybe! `/x` will still exist though, for being the marketing page for the advanced components. But if we were meaning to have one page especially for them in the paid version, I guess linking right to the pricing one might do the trick yes!